examples/Batch/WordCountFSharp/CSharpWorker.exe.config

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This file should be removed from the PR. The worker.exe and its config file should be from nuget.